content/projects/sanblas.md

```diff
@@ -0,0 +1,21 @@
+---
+title: San Blas
+repo: bensmithett/sanblas
+homepage: https://sanblas.netlify.com/
```

Comment:
```suggestion
homepage: https://sanblas.netlify.app/
```

Reply:
we've since migrated user sites to the `netlify.app` domain. know it's been some time, thanks - we're looking forward to merging this in once `homepage` update addressed!